### PR TITLE
Move the dart:html tour to webdev, update the web library overview

### PR DESCRIPTION
**Pull request by kwalrath**
Staged:

https://kw-webdev-dartlang-1.firebaseapp.com/guides/web-programming
https://kw-webdev-dartlang-1.firebaseapp.com/guides/html-library-tour

The code excerpts aren't been generated/handled correctly, but the pages look OK.

### CODE REVIEW

File: examples/html/lib/html.dart [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    // #enddocregion new-HttpRequest
+
+    // #docregion xxx
+    11111;
+    // #enddocregion xxx
+
+    // #docregion xxx
+    11111;
+    // #enddocregion xxx
+  }
```
[the remark this answers is not in the document]
Comment (chalin) [reply examples/html/lib/html.dart]: Fixed

**Comment by kwalrath** [comment]
Please ignore the cla:no label and merge this. Thanks for fixing this!

**Comment by kwalrath** [comment]
LGTM. Thanks!

**Comment by chalin** [comment]
@kwalrath - might you want to add an entry in this site's changelog for the changes introduced in this PR?

**Comment by chalin** [comment]
Also, there is no "Guides" entry in the sidenav. Should we consider adding one?

**Comment by kwalrath** [comment]
Yes, I should add that changelog entry. And we should add some kind of general web dev entry to the sidenav. I'll file bugs for these.

**Pull request merged**